### PR TITLE
Fix/buttons should not appear as links

### LINKED ISSUE #15358: Plain text appears to be a link
Plain text appears to be a link

-   **Severity:** Medium
-   **Affected Populations:**
    -   Low-Vision
    -   Motor Impaired
    -   Cognitively Impaired
-   **Platform(s):**
    -   All / Universal
-   **Components affected:**
    -   Publish and Unpublish

#### Issue description

There are three text elements within the Publish panel (the values for
"Visibility", "Publish", and "Suggestion") which are styled to
appear as links, but do not function as links. The entire containing
element is a button to open or close the expanding region, while the
text elements themselves have no direct function.

This may be confusing for users, since the expectation conveyed by their
appearance is that these elements are links.

##### Issue Code

```
    <button class="components-button components-panel__body-toggle" aria-expanded="false" type="button">


       ...


       Visibility:


       <span class="editor-post-publish-panel__link">Public</span>


    </button>





    <button class="components-button components-panel__body-toggle" aria-expanded="false" type="button">


        ...


        Publish:


        <span class="editor-post-publish-panel__link">Jan 18, 2019 4:40 pm</span>


    </button>





    <button class="components-button components-panel__body-toggle" aria-expanded="false" type="button">


       ...


        Suggestion:


        <span class="editor-post-publish-panel__link">Add tags</span></button>





    /* CSS */


    .editor-post-publish-panel__link {


        color: #007fac;


        text-decoration: underline;


        ...


    }
```

#### Remediation Guidance

Remove the `text-decoration` and `color` from these elements so that
they don't look like links.

##### Recommended Code

```
    <button class="components-button components-panel__body-toggle" aria-expanded="false" type="button">


       ...


       Visibility:


       <span class="editor-post-publish-panel__link">Public</span>


    </button>





    <button class="components-button components-panel__body-toggle" aria-expanded="false" type="button">


        ...


        Publish:


        <span class="editor-post-publish-panel__link">Jan 18, 2019 4:40 pm</span>


    </button>





    <button class="components-button components-panel__body-toggle" aria-expanded="false" type="button">


       ...


        Suggestion:


        <span class="editor-post-publish-panel__link">Add tags</span></button>





    /* CSS */


    .editor-post-publish-panel__link {


        ...


    }


```

#### Relevant standards

-   [1.3.3 Sensory Characteristics
    (Level A)](https://www.w3.org/TR/WCAG20/#content-structure-separation-understanding)

**Note:** This issue may be a duplicate with other existing accessibility-related bugs in this project. This issue comes from the Gutenberg accessibility audit, performed by [Tenon](https://www.tenon.io) and funded by [WP Campus](https://wpcampus.org/). This issue is GUT-13 in Tenon's report

### PR DESCRIPTION
**Pull request by getdave**
## Description
Closes https://github.com/WordPress/gutenberg/issues/15358#issuecomment-489615758.

Addresses issue from a11y report by removing link styling from buttons elements within the prepublish sidebar.

## How has this been tested?
* Open prepublish sidebar
* Verify the button text is not styled to visually appear as a link (eg: underlined and blue color)

## Screenshots 

Screen Shot 2019-05-06 at 15 42 55

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate.

### CODE REVIEW

File: packages/editor/src/components/post-publish-panel/style.scss [hunk 1]
```diff
@@ -54,10 +54,9 @@
 }
 
 .editor-post-publish-panel__link {
-	color: $blue-medium-700;
 	font-weight: 400;
 	padding-left: 4px;
-	text-decoration: underline;
+	text-decoration: none;
```
Comment (jorgefilipecosta) [inline packages/editor/src/components/post-publish-panel/style.scss]: Minor: I guess given that the element is a span, we don't need to add text-decoration: none; and we can just remove text-decoration: underline;.

**Review by jorgefilipecosta — APPROVED** [review]
Works as expected 👍

File: packages/editor/src/components/post-publish-panel/style.scss (hunk 1 above)
Comment (kjellr) [reply packages/editor/src/components/post-publish-panel/style.scss]: Agreed, this PR works fine with the text-decoration rule deleted. 👍

**Review by kjellr — APPROVED** [review]
This works for me! 

Screen Shot 2019-05-07 at 9 20 15 AM

I agree with @jorgefilipecosta's note about the `text-decoration` rule, but once that's removed this should be good to merge in. 👍 

Thanks!

**Comment by afercia** [comment]
Looks good from an a11y perspective, thanks for the PR! As already noted, `text-decoration: none;` seems redundant.

**Pull request merged**